.gitignore

```diff
@@ -1,8 +1,9 @@
+.idea/*
+.nyc_output
+build
 node_modules
-Makefile
+src/**.js
 coverage
-build
-./test
-.vscode
-settings.json
-**/*bundle*
+*.log
+package-lock.json
+dist
```

Comment:
Some files are missing an EOF new line. Not a big deal but it's kinda bugging me :smile: